src/handlers/payout/post.ts

```diff
@@ -84,160 +69,234 @@ export const incentivizeComments = async () => {
     }
     issueCommentsByUser[user.login].comments.push(issueComment.body_html);
   }
-  const tokenSymbol = await getTokenSymbol(paymentToken, rpc);
   logger.info(`Filtering by the user type done. commentsByUser: ${JSON.stringify(issueCommentsByUser)}`);
 
-  // The mapping between gh handle and comment with a permit url
-  const reward: Record<string, string> = {};
-
   // The mapping between gh handle and amount in ETH
   const fallbackReward: Record<string, Decimal> = {};
-  let comment = `#### Conversation Rewards\n`;
+
+  // array of awaiting permits to generate
+  const reward: { account: string; priceInEth: Decimal; userId: string; user: string; penaltyAmount: BigNumber }[] = [];
+
   for (const user of Object.keys(issueCommentsByUser)) {
     const commentsByUser = issueCommentsByUser[user];
     const commentsByNode = await parseComments(commentsByUser.comments, ItemsToExclude);
-    const rewardValue = calculateRewardValue(commentsByNode, incentives);
+    const rewardValue = calculateRewardValue(commentsByNode, calculateIncentives.incentives);
     if (rewardValue.equals(0)) {
       logger.info(`Skipping to generate a permit url because the reward value is 0. user: ${user}`);
       continue;
     }
     logger.debug(`Comment parsed for the user: ${user}. comments: ${JSON.stringify(commentsByNode)}, sum: ${rewardValue}`);
     const account = await getWalletAddress(user);
-    const amountInETH = rewardValue.mul(baseMultiplier);
-    if (amountInETH.gt(paymentPermitMaxPrice)) {
+    const priceInEth = rewardValue.mul(calculateIncentives.baseMultiplier);
```

Comment:
Perhaps we should rename this to `priceInBigNumber`

Reply:
I think we don't need to include `BigNumber` in the name because we have Typescript. `ETH` is wrong because it can technically be any token. maybe `rewardTokens`?

Reply:
Here `ETH` was meant to distinguish price in wei and price in ETH, not to mention a particular payment token

Reply:
I think we should think about changing to doing everything in wei because we will support multiple tokens and they can have different decimals so it doesn't make sense to do in ETH (18 decimals)